lisa/tools/ethtool.py

```diff
@@ -438,6 +438,25 @@ def _parse_rx_hash_level(self, interface: str, protocol: str, raw_str: str) -> N
         )
 
 
+class DeviceStatistics:
+    # NIC statistics:
+    #     tx_scattered: 0
+    #     tx_no_memory: 0
+    _statistics_pattern = re.compile(r"^\s+(?P<name>.*?)\: +?(?P<value>\d*?)\r?$")
+
+    def __init__(self, interface: str, device_statistics_raw: str) -> None:
+        self._parse_statistics_info(interface, device_statistics_raw)
+
+    def _parse_statistics_info(self, interface: str, raw_str: str) -> None:
+        statistics: Dict[str, int] = {}
+
+        items = find_groups_in_lines(raw_str, self._statistics_pattern)
+        statistics = {x["name"]: int(x["value"]) for x in items}
+
+        self.interface = interface
+        self.statistics = statistics
```

Comment:
`items`

Reply:
> `items`

sorry didn't get what you are trying to say?

Reply:
The variable name should be changed to `items`. Because it's in statistics already, the same name as the class doesn't make sense.

Reply:
I mean it doesn't need to repeat the `statistics` in both class and member names.

Reply:
> I mean it doesn't need to repeat the `statistics` in both class and member names.

hmm well thats how other classes also have in ethtool. But ok let me change it here.